yt/frontends/ramses/fields.py

```diff
@@ -95,82 +95,82 @@
 
 class RAMSESFieldInfo(FieldInfoContainer):
     known_other_fields: KnownFieldsT = (
-        ("Density", (rho_units, ["density"], None)),
-        ("x-velocity", (vel_units, ["velocity_x"], None)),
-        ("y-velocity", (vel_units, ["velocity_y"], None)),
-        ("z-velocity", (vel_units, ["velocity_z"], None)),
-        ("Pres_IR", (pressure_units, ["pres_IR", "pressure_IR"], None)),
-        ("Pressure", (pressure_units, ["pressure"], None)),
-        ("Metallicity", ("", ["metallicity"], None)),
-        ("HII", ("", ["H_p1_fraction"], None)),
-        ("HeII", ("", ["He_p1_fraction"], None)),
-        ("HeIII", ("", ["He_p2_fraction"], None)),
-        ("x-acceleration", (ra_units, ["acceleration_x"], None)),
-        ("y-acceleration", (ra_units, ["acceleration_y"], None)),
-        ("z-acceleration", (ra_units, ["acceleration_z"], None)),
-        ("Potential", (specific_ener_units, ["potential"], None)),
-        ("B_x_left", (b_units, ["magnetic_field_x_left"], None)),
-        ("B_x_right", (b_units, ["magnetic_field_x_right"], None)),
-        ("B_y_left", (b_units, ["magnetic_field_y_left"], None)),
-        ("B_y_right", (b_units, ["magnetic_field_y_right"], None)),
-        ("B_z_left", (b_units, ["magnetic_field_z_left"], None)),
-        ("B_z_right", (b_units, ["magnetic_field_z_right"], None)),
+        ("Density",                       (rho_units, ["density"], None)),
```

Comment:
FYI we have automatic-formatting tools enable that will actively undo this kind of manual formatting.